network/hostnetwork/future/handler_test.go

```diff
@@ -109,7 +109,7 @@ func TestPacketHandler_Handle_NotResponse(t *testing.T) {
 
 	ph.Handle(context.Background(), packet.NewReceivedPacket(resp, nil))
 
-	_, err := future.WaitResponse(time.Second)
+	_, err := future.WaitResponse(time.Millisecond)
```

Comment:
Could you please run it with 1200 count and check the test pass?

Reply:
Yes, I did
go test ./network/hostnetwork/future/... -run=TestPacketHandler_Handle_NotResponse  -count=20000
ok  	github.com/insolar/insolar/network/hostnetwork/future	26.482s